### ISSUE TITLE
Deep Learning Documentation Needed.

### ISSUE THREAD

**Issue by diffrxction** [dsc-tiu/open-source-ml-ai#8]
Need documentation on simple perceptron and their implementation from scratch. Code should not be copy pasted from online sources. Please attach relevant documentation with images in markdown files.

**Comment by ArmaanSeth** [comment]
Hi @diffrxction I would like to work on this on too. Let me know if I go ahead on this one.

Just to be sure by deep learning you mean Neural Networks.

**Comment by diffrxction** [comment]
> Hi @diffrxction I would like to work on this on too. Let me know if I go ahead on this one.
> 
> Just to be sure by deep learning you mean Neural Networks.

Sure @ArmaanSeth, feel free to go ahead and add code and documentation in the same format as earlier. 

You can start with single layer perceptrons and then move ahead to bigger neural networks as is convenient for you. For each different type of neural network, maintain a seperate subfolder for Python code and documentation. Thank you!

**Comment by ArmaanSeth** [comment]
@diffrxction ,Okay so I'll  make two code files one with single hidden layer(perceptron) and another with two hidden layers.

**Comment by ArmaanSeth** [comment]
@diffrxction  Hi just noticed that I have still not received the confirmation of the merge of my pull request. I am new at this can you help me out. Are there any changes that I have to do in my code/documentation, please let me know about that.
Regards
Armaan Seth

**Comment by diffrxction** [comment]
> @diffrxction Hi just noticed that I have still not received the confirmation of the merge of my pull request. I am new at this can you help me out. Are there any changes that I have to do in my code/documentation since the issue is still active, please let me know about that. Regards Armaan Seth

Which pull request @ArmaanSeth? This #18 ? I have already approved and merged this to the main branch of the repository. In case you are referring to a confirmation from Hacktoberfest then I guess you need to wait for 7 days before they can verify and validate your contribution. 

If your query is still not resolved then please reply to this with a comment containing the screenshot or more details about your problem.

Regards,
Aryan Pareek

**Comment by ArmaanSeth** [comment]
> > @diffrxction Hi just noticed that I have still not received the confirmation of the merge of my pull request. I am new at this can you help me out. Are there any changes that I have to do in my code/documentation since the issue is still active, please let me know about that. Regards Armaan Seth
> 
> Which pull request @ArmaanSeth? This #18 ? I have already approved and merged this to the main branch of the repository. In case you are referring to a confirmation from Hacktoberfest then I guess you need to wait for 7 days before they can verify and validate your contribution. 
> 
> If your query is still not resolved then please reply to this with a comment containing the screenshot or more details about your problem.
> 
> Regards,
> Aryan Pareek

Yes i I am talking about #18. For this i have not recieved the confirmation mail like I had received in case of Linear Regression. I have attached a screenshot of confirmation mail that I received in case of Linear Regression.![Screenshot_20221005-001421_Gmail.jpg](https://user-images.githubusercontent.com/99117431/193900737-bbbb7fe1-25f1-49ad-8584-78f78209eba7.jpg)

**Comment by diffrxction** [comment]
> Thanks for your help, my issue is now resolved , earlier this merge was not reflecting in hactoberfest page, it took a while but it appeared on the page Regards Armaan Seth

Glad to help!
